Tests/test_MarkovModel.py

```diff
@@ -17,11 +17,8 @@
 from io import StringIO
 
 try:
-    from numpy import array
-    from numpy import random  # missing in PyPy's micronumpy
-    from numpy import array_equal
-    from numpy import around
-    from numpy import log
```

Comment:
I don't think you needed to change any of these imports - maybe just revert all the changes to this file?